doc/spec/source/views/counted_view.rst

```diff
@@ -0,0 +1,24 @@
+.. SPDX-FileCopyrightText: Intel Corporation
+..
+.. SPDX-License-Identifier: BSD-3-Clause
+
+.. include:: ../include/distributed-ranges.rst
+
+.. _counted_view:
+
+=============
+ counted_view
+=============
+
+Interface
+=========
+
+Use `std::ranges::views::counted`_.
```

Comment:
We no longer support using std::ranges::views. There is always a `dr::mhp::views` or `dr::shp::views`. This is true for all views. I am not sure how to document it. Maybe enough to change
```suggestion
See `std::ranges::views::counted`_.
```